muschema/src/struct.ts

```diff
@@ -318,13 +320,13 @@ export class MuStruct<StructSpec extends _SchemaDictionary>
         // diffBinary subroutine
         const numProps = structProps.length;
         const trackerBytes = Math.ceil(numProps / 8);
-        methods.diffBinary.push(`var numPatch=0;var tracker=0;var trackerOffset = s.offset;s.grow(${trackerBytes});s.offset+=${trackerBytes};`);
+        methods.diffBinary.push(`var numPatch=0;var tracker=0;var trackerOffset = s.offset;s.grow(this.getByteLength(t)+${trackerBytes});s.offset+=${trackerBytes};`);
```

Comment:
we should include `trackerBytes` in the byte length of the struct.

Reply:
ah that's the plan, somehow missed that